src/core/localize.rkt

```diff
@@ -5,89 +5,84 @@
 
 (provide localize-error local-error-as-tree)
 
-(define (all-subexpressions expr)
-  (remove-duplicates
-   (reap [sow]
-         (let loop ([expr expr])
-           (sow expr)
-           (match expr
-             [(? number?) (void)]
-             [(? variable?) (void)]
-             [(list op args ...)
-              (for-each loop args)])))))
-
-; Compute local error or each sampled point at each node in `prog`.
-; Uses math/bigfloat rather than rival for speed.
-(define (compute-local-errors-fast prog ctx)
```

Comment:
Is it good to delete this unused code?

Reply:
Yes. Less code is better. We always have git to dig out old versions of the code. Be fearless about deleting unused code. Conveniently, while Racket has very few compile-time checks, it _does_ check that every name resolves. That's more than Python and JavaScript give you!